### PR TITLE
Reprovision

### PR DESCRIPTION
**Pull request by clarsonneur**

### CODE REVIEW

**Comment by clarsonneur** [comment]
@ojacques looks this LGTM?

File: vagrant/provision/roles/hubot/files/ninjabot/scripts/update.coffee [hunk 1]
```diff
@@ -31,35 +32,34 @@ module.exports = (robot) ->
     switch host
       when 'monitoring'
         server='monitoring-node'
-        command='sh ./provision.sh --limit=#{server} monitoringserver.yml'
```
Comment (ojacques) [inline vagrant/provision/roles/hubot/files/ninjabot/scripts/update.coffee:34]: 😁 need to be `./provision.sh -local --limit...` without this, there is an rsync which is not triggered in provision.sh, and we just reprovision the old version of the ansible playbook 😁

Comment (clarsonneur) [reply vagrant/provision/roles/hubot/files/ninjabot/scripts/update.coffee:34]: I'm wondering who missed that... :smile:

**Comment by clarsonneur** [comment]
:tada:

**Pull request merged**